src/Tes/Models/TesState.cs

```diff
@@ -11,11 +11,43 @@
  * Generated by: https://openapi-generator.tech
  */
 
+using System;
 using System.Runtime.Serialization;
 using Newtonsoft.Json;
 
 namespace Tes.Models
 {
+    public static class TesEnumUtility
+    {
+        // Static method to get the EnumMember Value using a switch statement
+        public static string GetEnumMemberValue(TesState state)
+        {
+            switch (state)
```

Comment:
This could be simplified to

```
return state switch
{
    TesState.UNKNOWNEnum => "UNKNOWN",
    TesState.QUEUEDEnum => "QUEUED",
    TesState.INITIALIZINGEnum => "INITIALIZING",
    TesState.RUNNINGEnum => "RUNNING",
    TesState.PAUSEDEnum => "PAUSED",
    TesState.COMPLETEEnum => "COMPLETE",
    TesState.EXECUTORERROREnum => "EXECUTOR_ERROR",
    TesState.SYSTEMERROREnum => "SYSTEM_ERROR",
    TesState.CANCELEDEnum => "CANCELED",
    _ => throw new ArgumentException("Invalid or unsupported enum value.", nameof(state))
};
```